spec/features/hiring_staff_can_visit_the_public_listings_spec.rb

```diff
@@ -85,13 +85,13 @@ def stub_accepted_terms_and_condition
   end
 
   def sign_in_user
-    click_on(I18n.t('nav.sign_in'))
+    first(:link, I18n.t('nav.sign_in')).click
```

Comment:
This is just a suggestion but you could also use the `within` method to execute an action _within_ a specific HTML element, for example: `within('.govuk-header__navigation.mobile-header-menu-item') { click_on(I18n.t('nav.sign_in')) }`

Reply:
Resolved in f59751c